.travis.yml

```diff
@@ -5,7 +5,5 @@ script:
     - go test -v ./...
```

Comment:
The idea was to keep the Travis file for the older versions, removing them, once we (gradually) arrive at the newer versions, so in this case you could just delete it. We are using GitHub Actions as the primary CI. GH actions had issues with older Go versions, that is why the travis file still existed.

Reply:
The fact that GH actions has issues with ancient Go versions, gives us a very strong hint that nobody is using them.

Reply:
You can remove the `.travis.yml` then